internal/authn/oidc/oidc.go

```diff
@@ -62,11 +62,13 @@ func (o *OAuth2Authenticator) Authenticate(ctx context.Context, t transport.Tran
 	u := &Claims{}
 	err = tok.Claims(u)
 	if err != nil {
+		log.Errorf("failed to extract claims: %v", err)
 		return nil, api.Deny
 	}
 
 	if o.EnforceAudCheck {
 		if u.Audience != o.CompletedConfig.ClientId {
+			log.Warnf("aud does not match the requesting client-id -- decision DENY")
```

Comment:
I wonder if those should be info or even debug :thinking:

Reply:
Yeah this one I wasn't 100% on. if you have `enforce-aud-check` on and this check fails, all youre going to get is an unauthorized error, it would be nice to know why it was unauthorized when it fails in this particular place. I didnt necessarily want to call it an error, but if its INFO, its not going to jump out in the logs very well with the hundreds of other INFO logs, WARN i thought was a good middle ground? I'm pretty flexible if we think it should be something else, but just providing my thinking if it helps.

Reply:
My only concern is that this is a problem with the user input, not a real error with the application itself or its configuration. It will correctly deny the access if the user is holding the wrong credentials / token.

Reply:
Fair enough, I'm thinking debug then as my main point for wanting the log here is so that the denial isn't hidden while testing if it's an aud check. If it's really only for testing debug is more reasonable. Sound good?

Reply:
It sounds good to me

Reply:
updated to use debug instead of warn.